build-coordinator/src/main/java/org/jboss/pnc/coordinator/builder/datastore/DatastoreAdapter.java

```diff
@@ -107,11 +107,8 @@ public void storeResult(BuildTask buildTask, BuildResult buildResult) throws Dat
                 return;
             }
 
-            // Build driver results
-            buildRecordBuilder.endTime(Date.from(Instant.now()));
```

Comment:
There is also duplication in the `initBuildRecordBuilder` itself.
I wonder if it makes sense to use `endTime` from the `buildTask` or just use `Instant.now` as when we store the result the operation is completed.